src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/arrivals/ConfirmationServiceTest.kt

```diff
@@ -97,16 +87,7 @@ class ConfirmationServiceTest {
   @Test
   fun `Confirm arrival matched to NOMIS offender who is in custody and is a non-court transfer, is rejected`() {
     whenever(prisonerSearchService.getPrisoner(any())).thenReturn(
-      PrisonerDetails(
-        firstName = FIRST_NAME,
-        lastName = LAST_NAME,
-        dateOfBirth = DATE_OF_BIRTH,
-        prisonNumber = PRISON_NUMBER,
-        pncNumber = null,
-        croNumber = CRO_NUMBER,
-        isCurrentPrisoner = true,
-        sex = GENDER
-      )
+      PRISONER_DETAILS_PROTOTYPE.copy(isCurrentPrisoner = true)
```

Comment:
tidied this up a bit by using the modify a prototype pattern to draw attention to aspects of the data that are relevant to the logic we're exercising